CONTRIBUTING.md

```diff
@@ -31,132 +33,166 @@ There are many ways in which you can contribute, beyond writing code. The goal o
   - [Attribution](#attribution)
 
 ## Asking Questions
+
 Have a question? Rather than opening an issue, please ask me directly on opensource@xfox111.net.
 
 ## Providing Feedback
+
 Your comments and feedback are welcome.
 You can leave your feedbak on feedback@xfox111.net or do it on [Microsoft Edge Add-ons Webstore](https://microsoftedge.microsoft.com/addons/detail/tabs-aside/kmnblllmalkiapkfknnlpobmjjdnlhnd), [Chrome Extensions Webstore](https://chrome.google.com/webstore/detail/tabs-aside/mgmjbodjgijnebfgohlnjkegdpbdjgin) or [Mozilla Add-ons Webstore](https://addons.mozilla.org/firefox/addon/ms-edge-tabs-aside/)
 
 ## Reporting Issues
+
 Have you identified a reproducible problem in the extension? Have a feature request? I'd like to hear it! Here's how you can make reporting your issue as effective as possible.
 
 ### Look For an Existing Issue
+
 Before you create a new issue, please do a search in [open issues](https://github.com/xfox111/TabsAsideExtension/issues) to see if the issue or feature request has already been filed.
 
 Be sure to scan through the [feature requests](https://github.com/XFox111/TabsAsideExtension/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement).
 
 If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment:
 
-* 👍 - upvote
-* 👎 - downvote
+- 👍 - upvote
```

Comment:
That is another "error" for Markdownlint

Reply:
Same situation. GitHub displays it correct. So, that means there's nothing to worry about. However, I've just checked the [official GitHub Markdown guidance article](https://guides.github.com/features/mastering-markdown/) and yeah, it features asterisks instead of dashes for bulletin lists. However, I still think that dashes are more readable for humans. Plus, it's dashes that we use to make a bulletin list in, say, `.txt` file. Besides, asterisks in MD are often used for bold and italic formatting, so I guess, there should be some diversity

Reply:
Should I revert changes or leave lists with dashes instead of asterisks?